candidate-standard/clause_8_collections.adoc

```diff
@@ -37,17 +37,19 @@ The OGC API-Common standard does not define any Information Resource types. Howe
 |/collections/{collectionId}/instances | Instances | List the available instances of the collection
 |===
 
-=== Parameter Modules
+==== Common query parameters
```

Comment:
Do we want to call this "common" Maybe "core" or "shared"?

Reply:
I prefer "shared" to avoid confusion of unintended references to OGC API - Common Part 1: Core.
I am a bear of very little brain ;-)